src/applications/vaos/tests/appointment-list/components/ConfirmedAppointmentDetailsPage/index.unit.spec.jsx

```diff
@@ -1029,7 +1029,7 @@ describe('VAOS <ConfirmedAppointmentDetailsPage>', () => {
   });
 });
 
-describe('VAOS <ConfirmedAppointmentDetailsPage> with VAOS service', () => {
+describe.skip('VAOS <ConfirmedAppointmentDetailsPage> with VAOS service', () => {
```

Comment:
These VAOS tests should be fixed in master

Reply:
Are all of the test failing?

Reply:
flaky test we set to skip a few days ago, a merge in from master should clear it out